docs/docs/aleph_zero/sdk.md

```diff
@@ -257,24 +236,101 @@ const sqrtPriceLimit = calculateSqrtPriceAfterSlippage(
 )
```

Comment:
1. `// slippage is a price change you are willing to accept`. Acceptable change between what? Acceptable deviation between the simulated price change as a result of swap at the time of preparing call and the actual price executed after the swap.
2. It should also be emphasized that this does not imply an acceptable slippage in terms of token amount - you can describe this in box